### PR TITLE
[ExecuTorch][Vulkan] Tests for et_vk.fused_ce

### PR DESCRIPTION
**Pull request by JCNTH**
Stack from [ghstack](https://github.com/ezyang/ghstack/tree/0.15.0) (oldest at bottom):
* #21005
* #20946
* #20945
* #20944
* #20943
* __->__ #20942
* #20941
* #20940
* #20939
* #20938
* #20937
* #20936
* #20935
* #20934
* #20933
* #20932
* #20931
* #20930
* #20929
* #20928
* #20927
* #20926
* #20925
* #20924
* #20923
* #20922
* #20921
* #20920
* #20919
* #20918
* #20917

Vulkan op-test golden for `et_vk.fused_ce` (ATen transcription of the CPU-eager loss/dlogits reference). 3 cases: all-valid, a masked label (`label<0`), and vocab >> workers (strided per-row reduce). Wires `fused_ce_test` in `targets.bzl` + `CMakeLists.txt`.
@exported-using-ghexport

Differential Revision: [D111761774](https://our.internmc.facebook.com/intern/diff/D111761774/)

Differential Revision: [D111761774](https://our.internmc.facebook.com/intern/diff/D111761774)

### CODE REVIEW

**Review by SS-JIA — APPROVED** [review]
Review automatically exported from Phabricator review in Meta.
